docs/_posts/Ahmetemintek/2024-01-17-sbiobertresolve_rxnorm_augmented_en.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: model
+title: Sentence Entity Resolver for RxNorm (sbiobert_base_cased_mli embeddings)
+author: John Snow Labs
+name: sbiobertresolve_rxnorm_augmented
+date: 2024-01-17
+tags: [rxnorm, licensed, en, clinical, entity_resolution]
+task: Entity Resolution
+language: en
+edition: Healthcare NLP 5.2.1
+spark_version: 3.0
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps clinical entities and concepts (like drugs/ingredients) to RxNorm codes using `sbiobert_base_cased_mli` Sentence Bert Embeddings. It trained on the augmented version of the dataset which is used in previous RxNorm resolver models. Additionally, this model returns concept classes of the drugs in `all_k_aux_labels` column.
+
+## Predicted Entities
+
+`RxNorm Codes`, `Concept Classes`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/3.Clinical_Entity_Resolvers.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_rxnorm_augmented_en_5.2.1_3.0_1705502473156.zip){:.button.button-orange.button-orange-trans.arr.button-icon.hidden}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_rxnorm_augmented_en_5.2.1_3.0_1705502473156.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
```

Comment:
@Ahmetemintek  please use NER + ER pipeline here, and set case sensitive as False in Bert embeddings. As an example, you can check here: https://nlp.johnsnowlabs.com/2023/05/31/sbiobertresolve_icd10cm_augmented_en.html